Sources/Commands/SwiftTool.swift

```diff
@@ -104,11 +113,12 @@ private class ToolWorkspaceDelegate: WorkspaceDelegate {
     }
 
     func willCheckOut(repository: String, revision: String, at path: AbsolutePath) {
-        queue.async {
-            self.stdoutStream <<< "Resolving \(repository) at \(revision)"
+        // 👀 this is a quick operation and does not provide a whole lot of value to report
```

Comment:
The only consideration is if there's an error from that operation — in that case it would be nice to know what was going on.  In a future, better, logging-and-diagnostics support it would be good to track all activities, but only show them if they are considered significant or if they fail.  But for this immediate PR this might be the right tradeoff.

Reply:
in this case, its a fairly quick mechanical git operation which could fail but should be pretty obvious from the error itself.

Reply:
having said that, removing this means nothing will be printed about what branch is used when using branch dependencies, see https://github.com/apple/swift-package-manager/pull/3427#discussion_r618844640


for example, here is the output when using this on SwiftPM itself which uses a mix of semantic versions and branch dependencies:

```
❯ ~/code/swift/swift-package-manager/.build/debug/swift-package update
Updating https://github.com/apple/swift-llbuild.git
Updated https://github.com/apple/swift-llbuild.git (1.44s)
Updating https://github.com/apple/swift-tools-support-core.git
Updated https://github.com/apple/swift-tools-support-core.git (0.87s)
Updating https://github.com/apple/swift-driver.git
Updated https://github.com/apple/swift-driver.git (1.27s)
Updating https://github.com/apple/swift-argument-parser.git
Updating https://github.com/jpsim/Yams.git
Updating https://github.com/apple/swift-crypto.git
Updated https://github.com/apple/swift-crypto.git (1.17s)
Updated https://github.com/apple/swift-argument-parser.git (1.17s)
Updated https://github.com/jpsim/Yams.git (1.17s)
Computing version for https://github.com/apple/swift-argument-parser.git
Computed https://github.com/apple/swift-argument-parser.git at 0.3.2 (0.24s)
Computing version for https://github.com/apple/swift-crypto.git
Computed https://github.com/apple/swift-crypto.git at 1.1.6 (0.22s)
Computing version for https://github.com/jpsim/Yams.git
Computed https://github.com/jpsim/Yams.git at 4.0.6 (0.21s)
```